pkg/minikube/audit/logFile.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package audit
+
+import (
+	"fmt"
+	"os"
+
+	"k8s.io/minikube/pkg/minikube/localpath"
+	"k8s.io/minikube/pkg/minikube/out/register"
+)
+
+var (
+	file    *os.File
+	logPath string
+)
+
+// setLogFile sets the logPath and creates the log file if it doesn't exist.
+func setLogFile() error {
+	logPath = localpath.AuditLog()
```

Comment:
I don't think this needs to exist as a global. Instead, it could be passed to `createLogFile`

Reply:
agree, that would make the funcs more unit-testable

Reply:
Updated to pass it in